LibTest/js_interop/interop_type_A02_t02.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion When interacting with a JS value, you need to provide a Dart type
+/// for it. You can do this by either using or declaring an interop type.
+/// Interop types are either a "JS type" provided by Dart or an extension type
+/// wrapping an interop type.
+///
+/// @description Check that it is a compile-time error to declare an external
```

Comment:
This, however, should indeed be a compile-time error as it will surely crash later.